### PR TITLE
Update interested-technical-writers.md

### PR DESCRIPTION
**Pull request by r1t1**
Please add my changes

### CODE REVIEW

**Comment by r1t1** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogle%2Fseason-of-docs%2Fpull%2F32) for more info**.
@googlebot I signed in

**Comment by r1t1** [comment]
Please add my changes

File: 2021-participants/interested-technical-writers.md [hunk 1]
```diff
@@ -24,3 +24,4 @@ Mayur Patil | <ul><li>[Blogspot](http://dazzlingmayur.blogspot.com)</li><li>[Cha
...
 Mfonobong Umondia David | https://mfonobong.contactin.bio/ | User Guides, Training Docs, Beginner Tutorials, Product Design, UX Writing
 Jaskirat Singh | [Link to my all activities and portfolio](https://github.com/jaskiratsingh2000/Activities-of-Jaskirat) | I am an open source research strategist and tech writer, eveloping new, and improving existing, computer-based technologies, systems, and solutions in advanced open source ecosystems. I have worked with CHAOSS/The Linux Foundation in the Google Season of Docs 2020 and had developed the [CHAOSS Community Handbook](https://handbook.chaoss.community/community-handbook/) from the scratch. All my progress and reports can be found within [my this repository](https://github.com/jaskiratsingh2000/Google-Season-of-Docs)
+Priti Mhatre | [Resume](https://drive.google.com/file/d/16pgh5AasV2X-8r6FALANfuuc8e0BwEca/view?usp=sharing) |<li> **Experience:** User Guides,Online help, API documentation, end user documentation</li><br><li><b>Tools & Technologies</b> DITA, Java, JSON, XML, HTML, CSS, Markdown, Confluence, </li> <li>**Interests:** Machine Learning, Web Development, Data Science, Analytics, Cloud Technologies</li>
```
Comment (r1t1) [inline 2021-participants/interested-technical-writers.md:27]: Add @r1t1 to interested-technical-writers.md

**Comment by r1t1** [comment]
Please review and add my name to the interested-technical-writers.md file.

**Review by r1t1 — COMMENTED** [review]
Reviewed

**Comment by emckean** [comment]
Hi! I think you need to reply "I signed it" rather than "I signed in" to make the CLA check pass.

**Comment by emckean** [comment]
Closing this pull request as the 2021 program is over. The 2022 program will begin soon; you can add your information in the [2022 interested technical writers file](https://github.com/google/season-of-docs/blob/main/2022-participants/interested-technical-writers.md).

**Pull request closed without merging**